surefire-api/src/test/java/org/apache/maven/surefire/api/util/RunOrderCalculatorTest.java

```diff
@@ -59,4 +62,44 @@ public void testOrderTestClasses()
     {
 
     }
+
+    public void reverseAlphabeticalRunOrderTestClasses()
```

Comment:
Names of these tests are invalid, They must start with "test" for JUnit 3 to find and run them

Reply:
In your new branch, repurpose all reversealphabetical tests to become TESTORDER tests.